native/src/components/OpeningHours.tsx

```diff
@@ -82,7 +82,9 @@ const OpeningHours = ({
         <Content>
           {openingHours.map((entry, index) => (
             <OpeningEntry
-              key={`${weekdays[index]!}-OpeningEntry`}
+              key={`${weekdays[index]}-OpeningEntry`}
+              // always supplied by Moment
```

Comment:
This comment will be out dated very soon, as we are about to remove Moment, also it is only supplied if the index is correct and not "always". Please remove the comment.